core/driver/sources/src/test/scala/org/virtuslab/ideprobe/dependencies/ErrorValidatorTest.scala

```diff
@@ -0,0 +1,119 @@
+package org.virtuslab.ideprobe.dependencies
+
+import scala.io.Source.fromResource
+
+import org.junit.Assert._
+import org.junit.Test
+
+import org.virtuslab.ideprobe.Config
+import org.virtuslab.ideprobe.IntelliJFixture
+import org.virtuslab.ideprobe.config.CheckConfig
+import org.virtuslab.ideprobe.config.CheckConfig.ErrorConfig
+import org.virtuslab.ideprobe.protocol.IdeMessage
+import org.virtuslab.ideprobe.protocol.IdeMessage.Level
+import org.virtuslab.ideprobe.reporting.ErrorValidator
+
+class ErrorValidatorTest {
+  private val configRoot = "probe"
+  private val defaultConfigWithErrorsEnabled: Config = Config.fromClasspath("error_validator.conf")
+  private val defaultCheckConfigWithErrorsEnabled: CheckConfig =
+    IntelliJFixture.readIdeProbeConfig(defaultConfigWithErrorsEnabled, configRoot).driver.check
+
+  private val messageBusConnectionImplNPE = fromResource("ideError_1.txt").mkString
+  private val uastMetaLanguageNPE = fromResource("ideError_2.txt").mkString
+
+  private val errorIdeMessages: List[IdeMessage] = List(messageBusConnectionImplNPE, uastMetaLanguageNPE)
+    .map(content => IdeMessage(level = Level.Error, content, None))
```

Comment:
you don't really need named parameter for `level` but it would be useful for that `None`.